script/release-vm-images

```diff
@@ -44,19 +48,20 @@ usage: $0 [options] VERSION
 OPTIONS:
   -h            Show this message
   -k            Keep release directory
-  -b BUCKET     The S3 bucket to upload vagrant images to [default: `flynn`]
-  -d DOMAIN     The CloudFront domain [default: `dl.flynn.io`]
+  -b BUCKET     The S3 bucket to upload vagrant images to [default: 'flynn']
+  -d DOMAIN     The CloudFront domain [default: 'dl.flynn.io']
   -r DIR        Resume the release using DIR
+  -c CACHE      Directory to cache base VM images [default: '/tmp']
```

Comment:
It doesn't look like you are actually setting the default value of `cache`, I would expect to see the following somewhere

```bash
cache="${cache:-"/tmp"}"
```